targets/ramips-mt7621

```diff
@@ -66,10 +66,13 @@ device('xiaomi-mi-router-4a-gigabit-edition', 'xiaomi_mi-router-4a-gigabit', {
 	factory = false,
 })
 
-device('xiaomi-mi-router-3g-v2', 'xiaomi_mi-router-3g-v2', {
+device('xiaomi-mi-router-3g', 'xiaomi_mi-router-3g', {
 	factory = false,
 })
```

Comment:
FWIW, the filename should probably contain the v1 designator and the string the autoupdater is looking for should be added as a manifest-alias.

Reply:
I thought about patching it in OpenWrt, but would be fine with a downstream solution as well.